### PR TITLE
Migrate latitude and longitude to G5Updatable::Location

### PR DESCRIPTION
**Pull request by neilmarion**
This allows SS Corpsearch to have the ability to search locations by proximity (https://www.pivotaltracker.com/story/show/107269666). The geolocation gem that we used (Geocoder) requires a longitude and latitude attribute in the G5Updatable::Location model.

### CODE REVIEW

**Comment by riverblufftrail** [comment]
@jcstringer can you review this?

File: db/migrate/20151103043916_add_latitude_and_longitude_to_location.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddLatitudeAndLongitudeToLocation < ActiveRecord::Migration
+  def change
+    add_column :g5_updatable_locations, :latitude, :float
```
Comment (jcstringer) [inline db/migrate/20151103043916_add_latitude_and_longitude_to_location.rb]: If geocoder is going to query on these columns we should probably add indexes for them.

Comment (neilmarion) [inline db/migrate/20151103043916_add_latitude_and_longitude_to_location.rb]: Indexes are now added in the migration.

**Comment by jcstringer** [comment]
:ship:

**Pull request merged**